native/src/routes/landing/containers/LandingContainer.js

```diff
@@ -62,22 +62,18 @@ class LandingContainer extends React.Component<ContainerPropsType> {
     const path = `/${cityCode}/${language}`
     const key: string = generateKey()
 
+    navigation.navigate('CityContent')
+
     // $FlowFixMe newKey missing in typedef
-    const action: NavigationReplaceAction = StackActions.replace({
+    navigation.dispatch(StackActions.replace({
```

Comment:
What is the difference between dispath and nativate?

Reply:
Dispatch takes an action, while navigate is the function. Furthermore, dispatch can take any Stack/Navigation Action. In our case what we want is described [here](https://reactnavigation.org/docs/nesting-navigators#navigator-specific-methods-are-available-in-the-navigators-nested-inside).